docs/pages/development/develop-your-project.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Use development builds
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { Tab, Tabs } from '~/ui/components/Tabs';
+import { BoxLink } from '~/ui/components/BoxLink';
+
+Usually, creating a new native build from scratch takes long enough that you'll be tempted to switch tasks and lose your focus. However, with the development build installed on your device or an emulator/simulator, you won't have to wait for the native build process until you [change the underlying native code](#rebuild-a-development-build) that powers your app.
+
+## Start the development server
+
+To start developing, run the following command to start the development server:
+
+<Terminal cmd={['$ npx expo start --dev-client']} />
+
+To open the project inside your development client:
+
+- Press <kbd>a</kbd> or <kbd>i</kbd> keys to open your project on an Android Emulator or an iOS simulator.
+- On a physical device, scan the QR code from your system's camera or a QR code reader to open the project on your device.
+
+## The launcher screen
+
+If you launch the development build from your device's Home screen, you will see your launcher screen, which looks similar to the following:
+
+<ImageSpotlight
+  alt="The launcher screen of a development build"
+  src="/static/images/dev-client/launcher-screen.png"
+  style={{ maxWidth: 600 }}
+/>
+
+If a bundler is detected on your local network, or if you've signed in to an Expo account in both Expo CLI and your development build, you can connect to it directly from this screen. Otherwise, you can connect by scanning the QR code displayed by the Expo CLI.
+
+## Debugging
+
+When you need to, you can access the menu by pressing <kbd>Cmd ⌘</kbd> + <kbd>d</kbd> or <kbd>Ctrl</kbd> + <kbd>d</kbd> in Expo CLI or by shaking your phone or tablet. Here you'll be able to access all of the functions of your development build, any debugging functionality you need, or switch to a different version of your app.
+
+## Add error handling
```

Comment:
I'd make this section first, I think it's easy to feel like "I got it" before getting to this point, but that import will save you a lot of pain